src/Controller/IdeaController.php

```diff
@@ -148,10 +148,15 @@ public function show(
             return $this->redirectToRoute('idea_show', ['id' => $idea->getId()]);
         }
 
+        $channelIsCreatable = $idea->isChannelCreatable($totalSupporters);
+        $supporterNeeded = $idea->supporterNeeded();
+
         return $this->render('idea/show.html.twig', [
             'idea' => $idea,
             'totalSupporters' => $totalSupporters,
             'isMember' => $isMember,
+            'channelIsCreatable' => $channelIsCreatable,
+            'supporterNeeded' => $supporterNeeded,
```

Comment:
You already pass the idea to twig, why do you need to send these methods ? You can call them directly from twig